src/components/contact-govuk-one-login/contact-govuk-one-login-controller.ts

```diff
@@ -1,7 +1,29 @@
 import { Request, Response } from "express";
+import { isValidUrl } from "./../../utils/strings";
+import pino from "pino";
 
-export function contactGet(req: Request, res: Response): void {
-  const data = {};
+const CONTACT_ONE_LOGIN_TEMPLATE = "contact-govuk-one-login/index.njk";
+const logger = pino();
 
-  res.render("contact-govuk-one-login/index.njk", data);
+export function contactGet(req: Request, res: Response): void {
+  const fromUrl = getFromUrlAndSaveIt(req);
+  if (!fromUrl) {
+    logger.error(
```

Comment:
```suggestion
    logger.info(
```
Now we're not showing users the 404 page when the param isn't present, can you make this INFO level? I don't want to be tripping any alarms when things are working as we expect.

Reply:
I would argue it's still an error that we are gracefully handling. But unexpected and shouldn't happen. But i can see your point so will update it